modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/macros/APIBuilderUI.macro

```diff
@@ -100,6 +100,35 @@ definition {
 		Alert.viewSuccessMessage();
 	}
 
+	@summary = "Default summary"
+	macro addEndpoint(fieldPath = null, fieldDescription = null, schemaName = null) {
+		if (isSet(schemaName)) {
```

Comment:
we have a macro 'createAPIEndpoint' to create an API endpoint, could you check it?

Reply:
I needed to change this macro for my test - Fixed, Thanks!